src/TimeoutMigrationTool/ApplicationOptions.cs

```diff
@@ -26,5 +26,6 @@ public class ApplicationOptions
         public const string NHibernateSourceDialect = "dialect";
         public const string AsqTargetConnectionString = "target";
         public const string AsqDelayedDeliveryTableName = "delayedtablename";
+        public const string UseRabbitDelayInfrastructureVersion1 = "use-v1";
```

Comment:
Let's not have a dash in the parameter

```suggestion
        public const string UseRabbitDelayInfrastructureVersion1 = "useV1";
```